inc/define.php

```diff
@@ -313,8 +313,6 @@
                                                   'Project', 'ProjectTask', 'Reservation',
                                                   'SoftwareLicense', 'Ticket', 'User', 'SavedSearch_Alert');
 
-$CFG_GLPI["notificationmethods_types"]    = array('NotificationMail');
```

Comment:
It seems it has been introduced in 581ae9cd544694fd0271f407c09cefba7a5dfaa1; but it has never been used :)